.github/scripts/check_instance.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+PUBLIC_IP=$1
+start_time=$(date +%s)
+
+while [ "$(curl -s -o /dev/null -w ''%{http_code}'' --max-time 5 http://$PUBLIC_IP)" != "200" ]; do
```

Comment:
Why use the command `-o /dev/null` to capture output if you're going to send it to `/dev/null`? If you want to avoid having it printed in the terminal, the -s silent option is enough.

Reply:
Might be a copypasta from chatgpt 😅 Good catch